tests/posit/connect/test_users.py

```diff
@@ -354,9 +266,123 @@ def test_user_cant_setattr(self):
     def test_count(self):
         responses.get(
             "https://connect.example/__api__/v1/users",
-            json=load_mock("v1/users.json"),
+            json=load_mock("v1/users?page_number=1&page_size=500.jsonc"),
             match=[responses.matchers.query_param_matcher({"page_size": 1})],
         )
         con = Client(api_key="12345", url="https://connect.example/")
         count = con.users.count()
-        assert count == 1
+        assert count == 3
+
+
+class TestUsersFindOne:
+    @responses.activate
+    def test_default(self):
+        # validate first result returned
+        responses.get(
+            "https://connect.example/__api__/v1/users",
+            json=load_mock("v1/users?page_number=1&page_size=500.jsonc"),
+        )
+        con = Client(api_key="12345", url="https://connect.example/")
+        user = con.users.find_one()
+        assert user.username == "al"
+        assert len(responses.calls) == 1
+
+    @responses.activate
+    def test_params(self):
+        # validate input params are propagated to the query params
+        params = {"key1": "value1", "key2": "value2", "key3": "value3"}
+        responses.get(
+            "https://connect.example/__api__/v1/users",
+            match=[
+                responses.matchers.query_param_matcher(
+                    {"page_size": 500, "page_number": 1, **params}
+                )
+            ],
+            json=load_mock("v1/users?page_number=1&page_size=500.jsonc"),
+        )
+        con = Client(api_key="12345", url="https://connect.example/")
+        con.users.find_one(**params)
```

Comment:
Does this need an assertion?

Reply:
The `match` value above is asserted by `responses`. But, on second thought, an additional assertion on `calls == 1` is wise. I'll add that.